caql/Metrics.g4

```diff
@@ -0,0 +1,18 @@
+grammar Metrics;

```

Comment:
What's the relationship between Metrics.g4 and the CaQL grammar?

Reply:
metric.g4 is used to parse the metric which is included in caql expression

Reply:
if only parsing caql and no further actions, metric.g4 is useless